x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/helper.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get, pick, isEmpty } from 'lodash';
+import { Section as SectionType } from './sections';
+import { Transaction } from '../../../../typings/es_schemas/ui/Transaction';
+import { APMError } from '../../../../typings/es_schemas/ui/APMError';
+import { Span } from '../../../../typings/es_schemas/ui/Span';
+import { flattenObject, FlattenItems } from '../../../utils/flattenObject';
+
+type Item = Transaction | APMError | Span;
+
+export type MetadataItems = Array<SectionType & { data?: FlattenItems }>;
```

Comment:
Isn't this a section? I'm a little unsure on the naming between sections and items. I think we should avoid data because it's generic and doesn't indicate plural. Instead use `items` or similar

Reply:
Perhaps you should have two types: `MetadataSection` and `MetadataRow` ?